tests/pytest_tests/unit_tests_old/test_launch/test_launch.py

```diff
@@ -940,7 +940,12 @@ def test_resolve_agent_config(monkeypatch, runner):
                 f,
             )
         config, returned_api = _launch.resolve_agent_config(
-            None, None, -1, ["diff-queue"], None
+            entity=None,
+            project=None,
+            max_jobs=-1,
+            queues=["diff-queue"],
+            config=None,
+            verbosity=None,
```

Comment:
wait so it can be a None?

Reply:
because in the function you have `int` you need to decide what is the correct one here, should it be 0?

Reply:
Thanks, I got myself confused before. Yes, verbosity can be `None`, in which case it behaves the same as 0. I'll make the type Optional[int].